src/widget/woverview.h

```diff
@@ -20,6 +20,13 @@ class PlayerManager;
 class QDomNode;
 class SkinContext;
 
+enum class OverviewType {
+    Filtered,
+    HSV,
+    RGB,
+    Count,
+};
```

Comment:
can you move this into WOverview and make it a `Q_ENUM`?

Then you can remove the `Count` member (anti-pattern) and instead use `QMetaEnum::keyCount`. It will also be simpler in the QComboBox because the enum can be implicitly converted to QVariant.

Reply:
Oh, nice! Thanks.